AutomationMentoring.iml

```diff
@@ -7,5 +7,15 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
```

Comment:
`.iml` file should not be in git. This is IDEA project file, it should be in git ignore